src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -86,13 +88,19 @@ public void initDefaultCommand() {
 
     public Command driveNormal() {
         return new Command("driveNormal", this) {
+            private double triggerSpeed = 0;
+            private double thumbstickSpeed = 0;
 
             @Override
             protected void execute() {
-                drive.arcadeDrive(
-                        +Robot.driveController.getTriggerAxis(Hand.kRight)
-                                - Robot.driveController.getTriggerAxis(Hand.kLeft),
-                        Robot.driveController.getX(Hand.kLeft));
+                triggerSpeed = -Robot.driveController.getTriggerAxis(Hand.kRight)
+                        + Robot.driveController.getTriggerAxis(Hand.kLeft);
+                thumbstickSpeed = Robot.driveController.getX(Hand.kLeft);
+                if (robot.lift.isSpeedLimitHeight()) {
```

Comment:
The only need for storing `robot` is to access the lift - more specifically, to access a property of the lift. Can we use NetworkTables to pass that data instead?

Reply:
I had mentioned this as an alternative when we wrote this. After talking with @msoucy I agree `NetworkTables` is the better option. It decouples these subsystems and lets us set the value from the dashboard for testing.

Reply:
done